src/views/studio/EditVideoSheet/EditVideoForm/EditVideoForm.tsx

```diff
@@ -329,8 +331,15 @@ export const EditVideoForm: React.FC<EditVideoFormProps> = ({
               name="category"
               control={control}
               rules={requiredValidation('Video category')}
+              onFocus={() => {
+                categorySelectRef.current?.scrollIntoView({
+                  behavior: 'smooth',
+                  block: 'start',
+                })
```

Comment:
Let's move that block into some function like `handleFieldFocus` that can accept ref as an argument